examples/level.erl

```diff
@@ -168,7 +168,7 @@ path() ->
 
 path_next() ->
   fun (PrevPath, _) ->
-      ?LET(NextSteps, vector(20, step()), PrevPath ++ NextSteps)
```

Comment:
I think that this is the default indentation that Emacs gives to `fun`s, so perhaps it should not be changed.